c/go/pkg/objects/subprocess.go

```diff
@@ -0,0 +1,38 @@
+// Virt Blocks
+//
+// Copyright (C) 2019 Red Hat, Inc.
+//
+// This software is distributed under the terms of the MIT License.
+// See the LICENSE file in the top level directory for details.
+
+package objects
+
+import (
+	"github.com/virtblocks/virtblocks/go/native/pkg/subprocess"
+	"sync"
+)
+
+var SubprocessObjectsLock sync.RWMutex
+var SubprocessObjects = make([]*subprocess.Subprocess, 1)
```

Comment:
Neither `subprocessObjects` nor `subprocessObjectsLock` should be accessible from outside the package, so they need to start with a lower case letter.

Reply:
right, fixed